src/coreclr/jit/rangecheck.h

```diff
@@ -287,6 +307,21 @@ struct RangeOps
         }
     }
 
+    // Given a constant limit in "l1", do ">>" it to l2 and mutate "l2".
```

Comment:
This comment is confusing (I realize you mostly cut-and-paste it from above). For one thing, `l2` never is mutated (maybe the arguments should be `const Limit&`). Also, "do '>>' it to l2" doesn't make sense. And it seems weird that the operation is basically `l2 >> l1` instead of `l1 >> l2`.

Reply:
Good point, addressed